topics/kmm-plugin-releases.md

```diff
@@ -34,12 +34,34 @@ Release info
 Release highlights
 </th>
 <th>
-Compatible Kotlin plugin version
+Compatible Kotlin version
 </th>
 </tr>
 <tr>
 <td>
 
+**0.2.1**
+
+Released: February 15, 2021
+
+</td>
+<td>
+
+To use this version of the plugin use [Android Studio 4.2 Beta](https://developer.android.com/studio/preview).
+* Compatibility and infrastructure improvements.
```

Comment:
Compatibility with what?
Compatibility with Kotlin 1.4.30.
Infrastructure improvements deserve a separate line.

Reply:
>Compatibility with Kotlin 1.4.30.

Seems like it's better to delete it at all because we decided to remove the line "Support for Kotlin 1.4.30". I would delete this.

Reply:
OK